test/filter_conversion.jl

```diff
@@ -236,6 +236,11 @@ end
             @test filt(H^0, x) ≈ x
         end
     end
+    # test that ^ doesn't cause stack overflow
+    @test_nowarn PolynomialRatio([1.0], [2.0])^typemin(Int8)
+    @test_nowarn ZeroPoleGain([1], [2], 3)^typemin(Int8)
+    @test_nowarn Biquad(1:5...)^typemin(Int8)
+    @test_nowarn SecondOrderSections(Biquad(1:5...))^typemin(Int8)
```

Comment:
Can we check for the correct result for at least some of those? E.g. something like
```julia
let H = PolynomialRatio([1.0], [2.0])^typemin(Int8)
    @test coefb(H) == [2.0^128]
    @test coefa(H) == [1.0]
end
```
For the `Biquad`/`SecondOrderSections` cases, that might be unwieldy, but at least `PolynomialRatio` and `ZeroPoleGain` should be doable.